packages/app/src/app/pages/common/GithubIntegration/index.tsx

```diff
@@ -1,36 +1,36 @@
-import { inject, hooksObserver } from 'app/componentConnectors';
-import React from 'react';
-import GithubLogo from 'react-icons/lib/go/mark-github';
+import React, { FunctionComponent } from 'react';
+import GitHubLogo from 'react-icons/lib/go/mark-github';
+
 import { Integration } from 'app/components/Integration';
+import { useOvermind } from 'app/overmind';
 
 type Props = {
   small?: boolean;
-  store: any;
-  signals: any;
 };
-export const GithubIntegration = inject('store', 'signals')(
-  hooksObserver(
-    ({
-      small = false,
-      signals: { signInGithubClicked, signOutGithubIntegration },
-      store: {
-        isLoadingGithub,
-        user: {
-          integrations: { github },
-        },
+export const GithubIntegration: FunctionComponent<Props> = ({
+  small = false,
+}) => {
+  const {
+    actions: { signInGithubClicked, signOutGithubIntegration },
+    state: {
+      isLoadingGithub,
+      user: {
+        integrations: { github },
       },
-    }: Props) => (
-      <Integration
-        bgColor="#4078c0"
-        description={small ? 'Commits & PRs' : 'Committing & Pull Requests'}
-        Icon={GithubLogo}
-        loading={isLoadingGithub}
-        name="GitHub"
-        onSignIn={() => signInGithubClicked({ useExtraScopes: true })}
-        onSignOut={signOutGithubIntegration}
-        small={small}
-        userInfo={github}
-      />
-    )
-  )
-);
+    },
+  } = useOvermind();
+
+  return (
+    <Integration
+      bgColor="#4078c0"
+      description={small ? 'Commits & PRs' : 'Committing & Pull Requests'}
+      Icon={GitHubLogo}
+      loading={isLoadingGithub}
+      name="GitHub"
+      onSignIn={() => signInGithubClicked({ useExtraScopes: true })}
```

Comment:
@christianalfoni `signInGithubClicked` expect to get an argument, but got `{ useExtraScopes: true }` as argument.

Should I delete the argument or should I update the type? 🤔

Reply:
Looks like `useExtraScopes` isn't used (it's always passed with a `true` value), so will update it here.
https://github.com/codesandbox/codesandbox-client/blob/cb965d440eb569d569e97ded524d54d142f241ad/packages/app/src/app/overmind/actions.ts#L163-L167

Came across this one too btw, don't know it that one's still used? 🤔
https://github.com/codesandbox/codesandbox-client/blob/cb965d440eb569d569e97ded524d54d142f241ad/packages/app/src/app/store/actions.js#L290-L308